docs/release-notes/NuGet-5.10.md

```diff
@@ -0,0 +1,172 @@
+
+* PMUI initialization fails with exception during debug session - [#10559](https://github.com/NuGet/Home/issues/10559)
+
+* nuget restore results in a package integrity check failure on big endian system - [#10567](https://github.com/NuGet/Home/issues/10567)
+
+* Remove unreachable code calling EventTrigger.dll - [#10584](https://github.com/NuGet/Home/issues/10584)
+
+* FormatException is thrown instead of PackagingException - [#10595](https://github.com/NuGet/Home/issues/10595)
+
+* CPVM - Concurrency issues in the graph walking algorithm - [#10598](https://github.com/NuGet/Home/issues/10598)
+
+* Add PMC powershell version telemetry - [#10609](https://github.com/NuGet/Home/issues/10609)
+
+* Improve NuGetVersion sort performance - [#10611](https://github.com/NuGet/Home/issues/10611)
+
+* Trusted-signers Add has inconsistent arguments - [#10647](https://github.com/NuGet/Home/issues/10647)
+
+* Vs2019 v16.9.0: Switching tabs in NuGet Package Manager from "Updates" to "Installed" doesn't update the frame. - [#10654](https://github.com/NuGet/Home/issues/10654)
+
+* Remove the "v" from the version number in PMUI - [#10677](https://github.com/NuGet/Home/issues/10677)
+
+* INuGetProjectService.GetInstalledPackagesAsync throws before receiving CPS project system nomination - [#10681](https://github.com/NuGet/Home/issues/10681)
+
+* Embedded Icons cause Access Denied from source "Microsoft Visual Studio Offline Packages" on Browse tab - [#10687](https://github.com/NuGet/Home/issues/10687)
+
+* INuGetProjectService.GetInstalledPackagesAsync throws when MSBuildProjectExtensionsPath is not set - [#10739](https://github.com/NuGet/Home/issues/10739)
+
+* "dotnet nuget remove source nuget.org" doesn't work the first time - [#10745](https://github.com/NuGet/Home/issues/10745)
+
+* Nuget blocks a threadpool thread in an async method making a synchronous call to the UI thread - [#10775](https://github.com/NuGet/Home/issues/10775)
+
+* Tools -> Options -> NuGet Package Manager string is truncated - [#10779](https://github.com/NuGet/Home/issues/10779)
+
+* `PackageLoadContext.GetInstalledAndTransitivePackagesAsync` is dead code and hurting performance - [#10790](https://github.com/NuGet/Home/issues/10790)
+
+* Use embedded icon in NuGet SDK packages - [#10795](https://github.com/NuGet/Home/issues/10795)
+
+* Update the SPDX license list - [#10806](https://github.com/NuGet/Home/issues/10806)
+
+**[List of all issues fixed in this release - 5.10](https://app.zenhub.com/workspaces/nuget-client-team-55aec9a240305cf007585881/reports/release?release=Z2lkOi8vcmFwdG9yL1JlbGVhc2UvNTY2MTQ)**
+  
+**[List of commits in this release - 5.10.0](https://github.com/NuGet/NuGet.Client/compare/5.9.0.7134...5.10.0.7240)**
+  
+### Community contributions
+
+Thank you to all the contributors who helped make this NuGet release awesome!
+
+|Who|PRs|Issues|
```

Comment:
Who can help generate 5.10 community contributions? /cc @nkolev92 @zkat @zivkan

Reply:
Previously I've done it, and the procedure is to look at closed pull requests with the Community label, follow the linked issue, and check that it's in the 5.10 release. Repeat until you start finding community PRs that are linked to issues in the 5.9 release.

I'm guessing the release notes tool can be improved to automate it the other way: for all the issues it found in the release, check if it was closed by a pull request, and if so, if that PR has the Community label.

Reply:
All the issues listed below are marked as being in 5.9, not 5.10 😕

Reply:
Do we have milestones or labels for release? I can create the query to use for reference in the future so it's less manual. I can PR to the release notes tool too once I get an understanding of how to identify the ship vehicle with the `community` label!

Reply:
Looks like ZenHub has the releases, but we don't have the same on GitHub which makes it a bit more difficult.

Here's the list I came up with:

- https://github.com/NuGet/Home/issues/10342
- https://github.com/NuGet/Home/issues/10139
- https://github.com/NuGet/Home/issues/5706
- https://github.com/NuGet/Home/issues/5016
- https://github.com/NuGet/Home/issues/10598
- https://github.com/NuGet/Home/issues/10562

Any missing?

Reply:
All of those seem included except https://github.com/NuGet/Home/issues/10139, which is a change that's engineering only. 

I think the precedent we set in the past was that we'd include all community contributions, but I was not able to find the reference.

Reply:
I'll add the updated contributor table tomorrow.

Reply:
Added